pages/popgen.py

```diff
@@ -0,0 +1,101 @@
+import holoviews as hv
+import numpy as np
+import panel as pn
+from bokeh import events
+from bokeh.models import ColumnDataSource
+from bokeh.models import CustomJS
+from holoviews import opts
+
+
+def make_one_way_stats(ts, windows_trees, windows_count, span_normalise, mode):
+    mode = mode.lower().split()[0]
+    windows_count = windows_count
+    if windows_trees:
+        windows = "trees"
+        x_axis = ts.breakpoints(as_array=True)[:-1]
+    else:
+        windows = np.concatenate(
+            (
+                np.arange(0, ts.sequence_length, ts.sequence_length // windows_count),
```

Comment:
I get this error when I try to use a tiny ts with 2 trees:
ZeroDivisionError: float division by zero

Reply:
Ah, good catch.